client/my-app/src/app/repositories/page.tsx

```diff
@@ -0,0 +1,113 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+
+interface RepositoriesProps {
+  accessToken?: string;
+  username?: string;
+}
+
+const Repositories: React.FC<RepositoriesProps> = ({
+  accessToken = "",
+  username = "YASH-RAJ-HANS",
+}) => {
```

Comment:
passing props and simultaneously retrieving from local storage can cause confusion